local/configs/plugins.txt

```diff
@@ -4,5 +4,6 @@ google-compute-engine
 job-dsl
 metrics
 monitoring
+opentelemetry
```

Comment:
This is required to force the latest version of the plugin rather than using the existing one in the docker base image.